install.sh

```diff
@@ -182,3 +182,14 @@ echo crew >> .git/info/sparse-checkout
 git fetch origin master
 git reset --hard origin/master
 echo "Chromebrew installed successfully and package lists updated."
+
+#check LD_LIBRARY_PATH on x86_64
+case "$architecture" in
+"x86_64")
+  (echo $LD_LIBRARY_PATH | grep '/usr/local/lib[^6]\|/usr/local/lib$' > /dev/null) || cat << EOF
```

Comment:
But it is the actual program, crew, modfiying the path. Of course, no package should do that.
But given the problems ChromiumOS gives us, it seems reasonable for a program to add stuff to enviroment. If the crew gets delteted, the additional path won't hurt in any way.
@uberhacker `~/.bashrc` is automatically read every time a terminal gets opened, no need for further interactions.
@jam7 If someone really shares a ~/.bashrc between, he really knows what he is doing, I don't think this is recommended.
Why would you not want to add one single library path, if this is the only way to get your programs working?